### PR TITLE
chore: bump rolling-ops lib

### PR DESCRIPTION
**Pull request by marcoppenheimer**

### CODE REVIEW

File: lib/charms/tls_certificates_interface/v1/tls_certificates.py [hunk 1]
```diff
@@ -1279,10 +1279,10 @@ def _on_relation_changed(self, event: RelationChangedEvent) -> None:
         """
         relation = self.model.get_relation(self.relationship_name)
         if not relation:
-            logger.warning(f"No relation: {self.relationship_name}")
+            logger.debug(f"No relation: {self.relationship_name}")
```
Comment (zmraul) [inline lib/charms/tls_certificates_interface/v1/tls_certificates.py:1282]: Is this a change of the lib?

Comment (marcoppenheimer) [reply lib/charms/tls_certificates_interface/v1/tls_certificates.py:1282]: No, I patched it. I'm sick of the `update-status` warnings.

Comment (marcoppenheimer) [reply lib/charms/tls_certificates_interface/v1/tls_certificates.py:1282]: I think it's changed in the V2+ versions of the lib.

I had raised an Issue for this a while ago, but apparently you can't release any more `V1` versions, once `V2` exists, so we're stuck with those noisy logs, or we patch.

Comment (zmraul) [reply lib/charms/tls_certificates_interface/v1/tls_certificates.py:1282]: lol sounds good then

**Review by zmraul — APPROVED** [review]

**Review by deusebio — APPROVED** [review]

**Pull request merged**